plutus-benchmark/bitwise/test/9.6/8 queens.pir.golden

```diff
@@ -66,9 +47,17 @@
                mkCons {integer} d (goList ds))
   in
   let
-    !writeBits : bytestring -> List integer -> List Bool -> bytestring
-      = \(bs : bytestring) (ixes : List integer) (bits : List Bool) ->
-          writeBits bs (goList ixes) (goList bits)
+    !writeBits : bytestring -> List integer -> Bool -> bytestring
+      = \(bs : bytestring) (ixes : List integer) (bit : Bool) ->
+          writeBits
+            bs
+            (goList ixes)
+            (Bool_match
+               bit
+               {all dead. bool}
+               (/\dead -> True)
+               (/\dead -> False)
+               {all dead. dead})
```

Comment:
It looks like maybe we should have `writeBitsTrue` and `writeBitsFalse` provided in the library, so that the `Bool_match` logic is reduced away statically. Maybe it doesn't matter much though. 

What sucks here is that we need to convert SOP lists to built-in lists, Aiken folks won't need that and so we're at a disadvantage here.